std_msgs/Header header
DetectedObj[] detectedObjList
